src/connections.jl

```diff
@@ -32,6 +32,21 @@ const CONNECTION_PARAMETER_DEFAULTS = _connection_parameter_dict(
     connection_options=CONNECTION_OPTION_DEFAULTS
 )
 
+# https://www.postgresql.org/docs/10/libpq-connect.html#LIBPQ-PQCONNECTSTARTPARAMS
+const CONNECTION_STATUS_MESSAGES = DefaultDict("Connecting",
+    libpq_c.CONNECTION_OK => "Connection is ready",
+    libpq_c.CONNECTION_BAD => "Connection failed",
+    libpq_c.CONNECTION_STARTED => "Waiting for connection to be made",
+    libpq_c.CONNECTION_MADE => "Connection OK; waiting to send",
+    libpq_c.CONNECTION_AWAITING_RESPONSE => "Waiting for a response from the server",
+    libpq_c.CONNECTION_AUTH_OK => "Received authentication; waiting for backend start-up to finish",
+    libpq_c.CONNECTION_SETENV => "Negotiating environment-driven parameter settings",
+    libpq_c.CONNECTION_SSL_STARTUP => "Negotiating SSL encryption",
+    libpq_c.CONNECTION_CHECK_WRITABLE => "Checking if connection is able to handle write transactions",
+    libpq_c.CONNECTION_CONSUME => "Consuming any remaining response messages on connection",
+    libpq_c.CONNECTION_GSS_STARTUP => "Negotiating GSSAPI",
+)::DefaultDict{libpq_c.ConnStatusType, String}
```

Comment:
Why not:
```julia
const CONNECTION_STATUS_MESSAGES = DefaultDict{libpq_c.ConnStatusType, String}(...)
```

Reply:
That method doesn't exist, but I could do `DefaultDict{libpq_c.ConnStatusType, String, String}(...)`